_assets/js/skip-content-links.js

```diff
@@ -0,0 +1,17 @@
+//= require ./scroll-into-view.js
```

Comment:
I like that you nested this.  We can start breaking up the js bundles perhaps for performance down the road.